docs/_static/text_wrapping.css

```diff
@@ -0,0 +1,6 @@
+.wy-table-responsive table td {
+    white-space: normal !important;
```

Comment:
Is the `!important` required?

Reply:
honestly I don't know. I pulled it from the sphinx-argparse docs https://sphinx-argparse.readthedocs.io/en/stable/misc.html#text-wrapping-in-argument-tables